### DISCUSSION TITLE
Feature Request: Add a Player Volume interface to the main page

### DISCUSSION THREAD

**Discussion by bengthu** [music-assistant/support#2363] [Feature requests and ideas]
I'd like to set volumes on the different players in my speaker group and would like them to be available without clicking to reveal the volumes for each of the players. See mockup.
![musicAssistantScreenWithPlayerVolumes](https://github.com/music-assistant/support/assets/28264113/70056941-b4b1-4757-8a9f-ee5e41eb4c3b)

**Comment by OzGav** [comment]
This adds clutter to the UI. Given that the equivalent is available with one click on the speaker icon this is unlikely to be implemented but we will leave this here and see if it gets many upvotes.

**Reply by bengthu** [reply]
Two clicks to view and a third to move the volume. You could make it an option to show/not show.

**Reply by Thyraz** [reply]
It's only one click (on the volume button next to the slider ) for me to open the volume slider view for the current group.
Are you clicking on the speaker icon instead, so you need one more click?

**Comment by bengthu** [comment]
I created a web page hosted on my HA server with 2 iframes that does what I need - see screenshot.
![aaaa](https://github.com/music-assistant/hass-music-assistant/assets/28264113/caf31574-d9e0-4950-baf7-dffa21aebd0e)
